src/openrct2-ui/windows/LoadSave.cpp

```diff
@@ -717,7 +717,7 @@ static void window_loadsave_paint(rct_window* w, rct_drawpixelinfo* dpi)
 
     // Draw path text
     auto ft = Formatter::Common();
-    ft.Add<uintptr_t>(Platform::StrDecompToPrecomp(buffer));
```

Comment:
Why did you remove this call? It is there for a good reason.

Reply:
Hi. I've seen that you closed it now. I didn't see that there was a PR for this already. In regard of removing the call, if I recall correctly this funtion is purely a cast.

Reply:
Note that it lives in `Platform`. It means the implementation is OS-dependent. Take a look at the macOS implementation and you will see that is not purely a cast.